exercises/ex05-offchain-workers/offchain-call/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "pallet-offchain-call"
+version = "2.5.0"
+edition = "2021"
+authors = ["Mrisho Lukamba"]
+description = "https://github.com/MrishoLukamba/MrishoLukamba"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = [
+	"derive",
+] }
+scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
+frame-benchmarking = { version = "4.0.0-dev", default-features = false, optional = true, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.28" }
```

Comment:
In our tutorials we are using substrate on branch `polkadot-v0.9.22`.
You should use the same one